test/src/header_test.dart

```diff
@@ -0,0 +1,101 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter_calendar_carousel/src/calendar_header.dart';
```

Comment:
Could you use `show` statement to be more clear where imported styles are actually coming from?

Reply:
Sure no Problem.  I think that in each individual widget that uses a few styles I can do `show someStyle, someOtherStyle` 

And in the main file I can import `as DefaultStyles` and use it like `DefaultStyles.someStyle` I shoul've done that in the previous PR :sweat_smile:

Reply:
I can do this for the main file on a separate PR

Reply:
Fine. Then I'll resolve this.